bin/euca-deploy

```diff
@@ -14,13 +14,15 @@ if __name__ == '__main__':
     parser.add_argument('-e', '--environment', default='etc/environment.yml')
     parser.add_argument('-p', '--password', default='foobar')
     parser.add_argument('-b', '--branch', default='euca-4.1')
+    parser.add_argument('-C', '--cookbook', default='https://github.com/eucalyptus/eucalyptus-cookbook')
```

Comment:
Id rather not have both -C and -c, also --cookbook is not descriptive. lets make it only --cookbook-repo, that way argparse will still work with --cookbook but the help will be more clear